inst/codec_data/parcel/parcel.R

```diff
@@ -0,0 +1,101 @@
+if (tryCatch(read.dcf("DESCRIPTION")[1, "Package"] == "codec", finally = FALSE)) {
+  devtools::load_all()
+} else {
+  library(codec)
+}
+message("Using CoDEC, version ", packageVersion("codec"))
+library(dplyr)
+library(sf)
+
+cagis_parcels <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/cagis_parcels-v1.1.0/cagis_parcels-v1.1.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble() |>
+  select(parcel_id, centroid_lat, centroid_lon, land_use, condo_id, market_total_value, acreage, homestead, rental_registration)
+
+online_parcels <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/auditor_online_parcels-v0.2.0/auditor_online_parcels-v0.2.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble()
+
+property_code_enforcements <- dpkg::stow("https://github.com/geomarker-io/parcel/releases/download/property_code_enforcements-v1.0.0/property_code_enforcements-v1.0.0.parquet") |>
+  dpkg::read_dpkg() |>
+  tibble::as_tibble() |>
+  mutate(
+    parcel_id = stringr::str_sub(cagis_parcel_id, 2), 
+    parcel_id = stringr::str_pad(parcel_id, width = 13, side = "right", pad = "0"),
+      .keep = "unused") |>
+  filter(date >= "2014-01-01") |> # filter to last 10 years
+  group_by(parcel_id, .drop = FALSE) |>
+  summarize(violations = n())
+
+d <- 
+  left_join(cagis_parcels, online_parcels, by = "parcel_id") |>
+  left_join(property_code_enforcements, by = "parcel_id") |>
+  st_as_sf(coords = c("centroid_lon", "centroid_lat"), crs = 4326) |>
+  st_transform(st_crs(cincy::tract_tigris_2010)) |>
+  st_join(cincy::tract_tigris_2010) |>
+  st_drop_geometry()
+
+d_land_use <- 
+  d |>
+  # collapse land_use categories
+  mutate(land_use_collapsed = 
+    case_when(
+      land_use %in% c("apartment, 40+ units", "apartment, 4-19 units", "office / apartment over", "apartment, 20-39 units") ~ "apartment",
```

Comment:
add this to readme

- apartment: `apartment, 40+ units`, `apartment, 4-19 units`, `office / apartment over`, ....
- single_family_dwelling: `single family dwelling`

and so on...